ibm_catalog.json

```diff
@@ -0,0 +1,68 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-secrets-manager-iam-engine",
+      "label": "Secrets Manager IAM credentials engine module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "secrets",
+        "iam engine",
+        "iam credentials engine",
+        "secrets manager"
+      ],
+      "short_description": "This module configures an IAM credentials engine for an existing Secrets Manager instance.",
+      "long_description": "Use this module to configure a secrets manager IAM credentials engine for an existing secrets manager instance.\n\nThe module supports the following operations:\n\n- Creates a new Service ID\n- Assigns the new Service ID \"Editor\" role access for `iam-groups`\n- Assigns the new Service ID \"Operator\" role access for `iam-identity`. If user set the input variable add_service_id_creator_role to true, this module will also add \"Service ID creator\" role access for `iam-identity`. It is recommended to set it true if the service ID creation is disabled in the IAM settings.\n- Creates a new API key for the Service ID\n- Optionally creates a secrets group, if an existing one is not passed in.\n- Creates a new arbitrary secret in the Secret Group with the generated Service ID API key value. Because it is an arbitrary secret, the IAM engine does not create the key or manage its rotation.\n- Configures the Secret Manager instance with the IAM credentials engine using the Service ID API key.\n\n#### Usage\n```hcl\nmodule \"iam_secrets_engine\" {\n  source                               = \"https://cm.globalcatalog.cloud.ibm.com/api/v1-beta/offering/source?archive=tgz&kind=terraform&name=terraform-ibm-secrets-manager-iam-engine&version=X.X.X\"  # Replace \"X.X.X\" with a release version to lock into a specific release\n  resource_group_id                    = \"xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX\"\n  region                               = \"us-south\"\n  iam_engine_name                      = \"iam-engine\"\n  secrets_manager_guid                 = \"XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX\"\n}\n```",
```

Comment:
I think move the `# Replace "X.X.X" with a release version to lock into a specific release to the line above source, because the source is so long, it wont be seen